benchmarks/db/db.py

```
@@ -0,0 +1,94 @@
+        # We are allowing the single-object array
+        # e.g. [{'id':5, 'randomNumber':10}] for now,
+        # but will likely make this fail at some point
+        if type(response) == list:
+            response = response[0]
+            problems.append((
+                'warn',
+                'Response is a JSON array. Expected JSON object (e.g. [] vs {})',
+                url))
+
+            # Make sure there was a JSON object inside the array
+            if type(response) != dict:
```

Ruff

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)